versions.txt

```diff
@@ -1,48 +1,137 @@
 # Format:
 # module:released-version:current-version
-google-cloud-appengineflexcompat:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-appengineflexcustom:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-appengineflexjava:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-appenginejava8:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-bigquery:1.27.0:1.27.1-SNAPSHOT
-google-cloud-bigquerydatatransfer:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-bigtable:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-bom:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-compat-checker:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-compute:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-container:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-contrib:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-core:1.27.0:1.27.1-SNAPSHOT
-google-cloud-core-grpc:1.27.0:1.27.1-SNAPSHOT
-google-cloud-core-http:1.27.0:1.27.1-SNAPSHOT
-google-cloud-dataproc:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-datastore:1.27.0:1.27.1-SNAPSHOT
-google-cloud-dialogflow:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-dlp:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-dns:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-errorreporting:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-examples:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-firestore:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-language:1.27.0:1.27.1-SNAPSHOT
-google-cloud-logging:1.27.0:1.27.1-SNAPSHOT
-google-cloud-logging-logback:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-managedtest:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-monitoring:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-nio:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-nio-examples:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-notification:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-os-login:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-pom:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-pubsub:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-resourcemanager:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-spanner:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-speech:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-storage:1.27.0:1.27.1-SNAPSHOT
-google-cloud-texttospeech:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-testing:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-trace:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-translate:1.27.0:1.27.1-SNAPSHOT
-google-cloud-util:0.45.0-alpha:0.45.1-alpha-SNAPSHOT
-google-cloud-video-intelligence:0.45.0-beta:0.45.1-beta-SNAPSHOT
-google-cloud-vision:1.27.0:1.27.1-SNAPSHOT
+
+#
+# common versions
+#
+#grpc-google-common-protos:1.9.0:1.9.0
+#grpc-google-iam-v1:0.10.0:0.10.0
+#proto-google-common-protos:1.9.0:1.9.0
+#proto-google-iam-v1:0.10.0:0.10.0
+
+#
+# google-api-pom versions
+#
+# parent
+google-api-pom:0.11.0:0.11.0
+# children
+#grpc-google-cloud-datastore-v1
+grpc-google-cloud-bigquerydatatransfer-v1:0.11.0:0.11.0
```

Comment:
It seems like the logical order would be to put the grpc packages after the proto ones

Reply:
Well, my idea was to put them in order of "release". The more basic packages are on top